src/migrations/tasks/20230315112325_service-areas-table.ts

```diff
@@ -0,0 +1,107 @@
+import { makeDomainLogger } from "back-end/lib/logger";
+import { console as consoleAdapter } from "back-end/lib/logger/adapters";
+import Knex from "knex";
+
+const logger = makeDomainLogger(consoleAdapter, "migrations");
+
+enum TWUServiceArea {
+  Developer = "DEVELOPER",
+  DataSpecialist = "DATA_SPECIALIST",
+  ScrumMaster = "SCRUM_MASTER",
+  DevopsSpecialist = "DEVOPS_SPECIALIST"
+}
```

Comment:
Any good reason why the 'name' column in the ServiceArea table couldn't contain spaces/be user readable? Could flip the keys and values around to accommodate. If we wanted Human Readable labels without string manipulation on the FE, this would be one way to achieve that. 
```
enum TWUServiceArea {
  DEVELOPER = "Developer",
  DATA_SPECIALIST = "Data Specialist",
  SCRUM_MASTER = "Scrum Master",
  DEVOPS_SPECIALIST = "Dev Ops Specialist"
}
```